pkg/controller/controller_test.go

```diff
@@ -334,136 +341,136 @@ func TestDeleteVolumeSnapshot(t *testing.T) {
-	}
-
-	result := c.DeleteReplication(context.NewAdminContext(), req)
-	if result != nil {
-		t.Errorf("Expected %v, got %v\n", nil, result)
-	}
-}
-
-func TestEnableReplication(t *testing.T) {
-	mockClient := new(dbtest.MockClient)
-	mockClient.On("GetProfile", context.NewAdminContext(), "1106b972-66ef-11e7-b172-db03f3689c9c").Return(&SampleProfiles[0], nil)
-	mockClient.On("GetDockByPoolId", context.NewAdminContext(), "084bf71e-a102-11e7-88a8-e31fe6d52248").Return(&SampleDocks[0], nil)
-	db.C = mockClient
-
-	var req = &model.ReplicationSpec{
-		BaseModel: &model.BaseModel{
-			Id: "bd5b12a8-a101-11e7-941e-d77981b584d8",
-		},
-		ProfileId: "1106b972-66ef-11e7-b172-db03f3689c9c",
-		PoolId:    "084bf71e-a102-11e7-88a8-e31fe6d52248",
-	}
-	var c = &Controller{
-		volumeController: NewFakeVolumeController(),
-	}
-
-	result := c.EnableReplication(context.NewAdminContext(), req)
-	if result != nil {
-		t.Errorf("Expected %v, got %v\n", nil, result)
-	}
-}
-
-func TestDisableReplication(t *testing.T) {
-	mockClient := new(dbtest.MockClient)
-	mockClient.On("GetProfile", context.NewAdminContext(), "1106b972-66ef-11e7-b172-db03f3689c9c").Return(&SampleProfiles[0], nil)
-	mockClient.On("GetDockByPoolId", context.NewAdminContext(), "084bf71e-a102-11e7-88a8-e31fe6d52248").Return(&SampleDocks[0], nil)
-	db.C = mockClient
-
-	var req = &model.ReplicationSpec{
-		BaseModel: &model.BaseModel{
-			Id: "bd5b12a8-a101-11e7-941e-d77981b584d8",
-		},
-		ProfileId: "1106b972-66ef-11e7-b172-db03f3689c9c",
-		PoolId:    "084bf71e-a102-11e7-88a8-e31fe6d52248",
-	}
-	var c = &Controller{
-		volumeController: NewFakeVolumeController(),
-	}
-
-	result := c.DisableReplication(context.NewAdminContext(), req)
-	if result != nil {
-		t.Errorf("Expected %v, got %v\n", nil, result)
-	}
-}
-
-func TestFailoverReplication(t *testing.T) {
-	mockClient := new(dbtest.MockClient)
-	mockClient.On("GetProfile", context.NewAdminContext(), "1106b972-66ef-11e7-b172-db03f3689c9c").Return(&SampleProfiles[0], nil)
-	mockClient.On("GetDockByPoolId", context.NewAdminContext(), "084bf71e-a102-11e7-88a8-e31fe6d52248").Return(&SampleDocks[0], nil)
-	db.C = mockClient
-
-	var vol = &model.ReplicationSpec{
-		BaseModel: &model.BaseModel{
-			Id: "bd5b12a8-a101-11e7-941e-d77981b584d8",
-		},
-		ProfileId: "1106b972-66ef-11e7-b172-db03f3689c9c",
-		PoolId:    "084bf71e-a102-11e7-88a8-e31fe6d52248",
-	}
-
-	var failover = &model.FailoverReplicationSpec{Failover: struct {
-		AllowAttachedVolume bool   `json:"allowAttachedVolume,omitempty"`
-		SecondaryBackendId  string `json:"secondaryBackendId,omitempty"`
-	}{AllowAttachedVolume: true, SecondaryBackendId: "084bf71e-a102-11e7-88a8-e31fe6d52249"}}
-	var c = &Controller{
-		volumeController: NewFakeVolumeController(),
-	}
-
-	result := c.FailoverReplication(context.NewAdminContext(), vol, failover)
-	if result != nil {
-		t.Errorf("Expected %v, got %v\n", nil, result)
-	}
-}
+//func TestCreateReplication(t *testing.T) {
```

Comment:
Is there a reason to keep this commented out in the code?

Reply:
If there is a reason to keep these commented out tests, please added a NOTE to explain why.  Otherwise, they should be removed now and added back later when they are ready.

Reply:
okay, I will remove it.